libsrc/cbm/tgi_lo.inc

```diff
@@ -129,6 +135,24 @@ INSTALL:
 UNINSTALL:
         rts
 
+; Runtime PET patch
```

Comment:
Likely I'm missing the point but wouldn't it be more appropriate to just use `.if .def(__PET__)` in SETPIXEL ?

Reply:
As it stands the driver source is assembled 8 times to build 8 drivers. But you consider it important that they are identical instead of saying "if we build the driver 8 times anyway we can as well make them slightly different". You surely understand that this seems somewhat strange to me, don't you ;-)

If it is however important to you to showcase that the same file works for all CBMs I propose you this "compromise": Keep the runtime patch, but .ifdef it with PET. Then you know that you can use the PET driver for all CBMs while the usual user has a smaller driver for the only target he's interested in (apart from the PET).